pkg/server/test/read.go

```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/openfga/openfga/pkg/typesystem"
 )
 
+// Read Command delegates to [storage.ReadPage].
+// TODO Tests here shouldn't assert on correctness of results because that should be tested in pkg/storage/test.
```

Comment:
Mind if we move the todo out and create a linked issue instead?

Reply:
I'll keep the comment and I created a JIRA internally